pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -221,7 +225,8 @@ let
     name = "bazel-build-${pname}-${version}";
 
     # See https://github.com/google/jax/blob/main/.bazelversion for the latest.
-    bazel = bazel_6;
+    #bazel = bazel_6;
+    bazel = bazel;
```

Comment:
This causes infinite recursion (and breaks nixpkgs evaluation), right?

Reply:
yep, eval broken

Reply:
The package is marked as `broken` and CI eval was happy; did this actually break the branch? Apologies for missing it, though.